sgl-kernel/csrc/cpu/gemm_fp8.cpp

```diff
@@ -0,0 +1,356 @@
+#include "common.h"
+#include "vec.h"
+#include "gemm.h"
+
+namespace {
+
+template <typename scalar_t>
+inline void copy_stub(scalar_t* __restrict__ out, const float* __restrict__ input, int64_t size) {
+  using bVec = at::vec::Vectorized<scalar_t>;
+  using fVec = at::vec::Vectorized<float>;
+  constexpr int kVecSize = bVec::size();
+
+  int64_t d;
+  #pragma GCC unroll 4
```

Comment:
for code formatting, follow: https://docs.sglang.ai/references/contribution_guide.html#code-formatting-with-pre-commit

Reply:
Formatted the code via `pre-commit run --all-files` in https://github.com/mingfeima/sglang/pull/47/commits/23b4e62a115dc81165ed1211e52474ce0268302b